### PR TITLE
bot: add cluster scan help docs

### PR DESCRIPTION
**Pull request by vaijab**
## Description

Part of https://github.com/kubeshop/botkube-cloud/issues/1062

Changes proposed in this pull request:

- bot: add cluster scan help docs

### CODE REVIEW

File: pkg/bot/interactive/help.go [hunk 1]
```diff
@@ -209,10 +209,11 @@ func (h *HelpMessage) aiPlugin() []api.Section {
 		{
 			Base: api.Base{
 				Header:      "🤖 AI powered Kubernetes assistant",
-				Description: fmt.Sprintf("`%s ai` use natural language to ask any questions", api.MessageBotNamePlaceholder),
+				Description: fmt.Sprintf("`%s ai` use natural language to ask any questions\n`%s ai scan` perform a cluster-wide scan for issues", api.MessageBotNamePlaceholder, api.MessageBotNamePlaceholder),
 			},
 			Buttons: []api.Button{
 				h.btnBuilder.ForCommandWithoutDesc("Ask a question", "ai hi!", api.ButtonStylePrimary),
+				h.btnBuilder.ForCommandWithoutDesc("Cluster Scan", "ai scan", api.ButtonStylePrimary),
```
Comment (pkosiec) [inline pkg/bot/interactive/help.go:216]: Please double check with @miklosp if we should have these two lines. IMO the `Ask a question` button is quite unnecessary and as now we have a very useful command, we could keep a single button.

Comment (vaijab) [reply pkg/bot/interactive/help.go:216]: @miklosp please advise.

Comment (vaijab) [reply pkg/bot/interactive/help.go]: image
updated with suggested changes.

**Review by pkosiec — APPROVED** [review]
LGTM, but I'd wait for @miklosp for final confirmation 👍 

image

**Pull request merged**